src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react'
+import { Box, Typography } from '@mui/material'
+import QuizQuestion from '~/containers/my-quizzes//scroll-question/ScrollQuestions'
+
+import { QuizViewProps } from '~/types'
+import { styles } from '~/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles'
+
+const ScrollQuestionsQuizView: FC<QuizViewProps> = ({ questions }) => {
+  const questionList = questions.map((question, index) => (
+    <QuizQuestion index={index} key={question._id} question={question} />
+  ))
+
+  return (
+    <Box>
+      <Typography sx={styles.description}>
+        Lorem ipsum dolor sit amet consectetur. Elementum at diam vulputate
+        suspendisse nulla tortor tempor odio tortor. Eu eleifend quam eget et
+        nulla.
```

Comment:
Maybe you should move this test to translations, what do you think?

Reply:
> Maybe you should move this test to translations, what do you think?

Is it possible translate Lorem ipsum text ? It's temporary here according to design

Reply:
Yes, but in the future we will change this to the correct text. You can move this text to 'en' for now

Reply:
> Yes, but in the future we will change this to the correct text. You can move this text to 'en' for now

When we change the text and only then we can add it to 'en'. 
I suppose it is unnecessary to add random text to 'en'

Reply:
Maybe you are right.